frontends/mit-open/src/pages/FieldPage/FieldPageSkeleton.tsx

```diff
@@ -311,6 +311,7 @@ const FieldSkeletonProps: React.FC<FieldSkeletonProps> = ({
             <FeaturedCoursesCarousel
               title="Featured Courses"
               config={FEATURED_RESOURCES_CAROUSEL}
+              isLoading={field.isLoading}
```

Comment:
The carousel normally displays a loading state while the API call specified by its config is in flight.

Adding `isLoading` delays any API call from being initiated until `true`. In this case, after the field has loaded.